NeosModLoader/NeosMod.cs

```diff
@@ -6,6 +6,8 @@ namespace NeosModLoader
     // contains members that only the modloader or the mod itself are intended to access
     public abstract class NeosMod : NeosModBase
     {
+        public static bool IsDebugEnabled() => Logger.IsDebugEnabled();
+        public static void DebugFunc(Func<string> messageProducer) => Logger.DebugFuncExternal(messageProducer);
```

Comment:
Also do we really want to publicly expose a debugging function that accepts function?

Reply:
I mean, I think that this is much more usable

```cs
DebugFunc(() => $"foo bar {ExpensiveComputation()}");
```

...than the non-function alternative, which is:
```cs
if (DebugEnabled())
{
    Debug($"foo bar {ExpensiveComputation()}");
}
```

What's your concern with passing a function?

Reply:
Oh right I see now, it doesn't get called at all if debugging isn't enabled.